documentation/en/source/pages/templates-spec/builders/builders-azure.rst

```diff
@@ -49,8 +49,197 @@ For building an image, the valid keys are:
 
 .. note:: When building from a scan, your yaml or json file must contain an ``installation`` section in ``builders``. This is mandatory when you create a new template, but might be missing when you build from a scan. Make sure it is present or your build will fail.
 
-Publishing a Machine Image
---------------------------
+
+
+Publishing a Machine Image with Azure Resource Manager Connector
+----------------------------------------------------------------
+
+To publish an image, the valid keys are:
+
+* ``type`` (mandatory): a string providing the machine image type to build. Default builder type for Azure: ``Microsoft Azure``. To get the available builder type, please refer to :ref:`command-line-format`
+* ``account`` (mandatory): an object providing all the cloud account information to authenticate and publish a machine image to Azure.
+* ``storageAccount`` (mandatory): a string providing the storage account to use for uploading and storing the machine image. The storage account is the highest level of the namespace for accessing each of the fundamental services.
+* ``container`` (mandatory): the container that will contains the blob file in Azure Cloud. If the names provides doesn't already exist it will be created.
```

Comment:
A few typos correct to :=> the container that will contain the blob file in Azure Cloud. If the name provided does not already exist it will be created.